### DISCUSSION TITLE
minio local server

### DISCUSSION THREAD

**Discussion by willianmacedo** [minio/minio#20086] [Q&A]
Hi guys,
I'm a beginner at least and would like your help with a local installation. I'm using Cloudflare's tunnel to point to my server. to access I used cloudflare tunnels

8023
miniofront.mydomain.com.br 

8024
minioback.mydomain.com.br

I'm using this stack

version: '3.8'
services:
  minio:
    image: quay.io/minio/minio
    volumes:
      - ./data:/data
    ports:
      - 8024:9000
      - 8023:9001
    environment:
      MINIO_ROOT_USER: 'user'
      MINIO_ROOT_PASSWORD: '1234'
      MINIO_ADDRESS: ':9000'
      MINIO_CONSOLE_ADDRESS: ':9001'
    command: minio server /data

**Comment by jpacora** [comment]
You should set the env vars for MINIO_SERVER_URL, MINIO_BROWSER_REDIRECT_URL. I can help you if you want 😜

**Reply by willianmacedo** [reply]
Thank you very much, I'm lost and have no way out, do you need access?

**Reply by jpacora** [reply]
Sure, right now I'm having a issue with MinIO and cloudlfare tunnel (#20087), so if we setup your instance I can test if is a "CloudFlare-MinIO" issue 😜

**Reply by jpacora** [reply]
Delete your comment for security purposes. Can we chat or email?

**Reply by willianmacedo** [reply]
yes. willianautocom@gmail.com

**Reply by willianmacedo** [reply]
I deleted the mini server stack

**Reply by jpacora** [reply]
Email sent!

**Comment by willianmacedo** [comment]
hello
I ended up deleting the stacks by accident, I was testing with bkp from
proxmox.

Em ter., 16 de jul. de 2024 às 15:19, Jorge Pacora ***@***.***>
escreveu:

**Comment by willianmacedo** [comment]
You were successful in the tests? I will reactivate the user

portainer.datascale.com.br
admin
portainer1234

minio.datascale.com.br
admin
minio1234

s3.datascale.com.br

stack

version: '3.8'
services:
  minio:
    image: quay.io/minio/minio
    volumes:
      - ./data:/data
    ports:
      - 8024:9000
      - 8023:9001
    environment:
      MINIO_ROOT_USER: 'admin'
      MINIO_ROOT_PASSWORD: 'minio1234'
      MINIO_ADDRESS: ':9000'
      MINIO_CONSOLE_ADDRESS: ':9001'
    command: minio server /data



Em ter., 16 de jul. de 2024 às 15:19, Jorge Pacora ***@***.***>
escreveu:

**Comment by willianmacedo** [comment]
How do I know that the buckets are working correctly on s3.datascale.com.br?
How can I test?

Em sex., 19 de jul. de 2024 às 10:08, willian macedo <
***@***.***> escreveu:
